python/datasources/covid_tracking_project.py

```diff
@@ -14,5 +17,112 @@ def get_id():
     def get_table_name():
         return 'covid_tracking_project'
 
+    @staticmethod
+    def get_standard_columns():
```

Comment:
Are you still planning on splitting `race_and_ethnicity` into two columns (`race` and `includes_hispanic`)? I wasn't sure if that was intended as a followup or part of this

Reply:
Yes, when the metadata is joined the includes_hispanic column is added. I used the column name race_and_ethnicity still because it is a standard across multiple data sets, and it will still include ethnicity alone as its own "race category" (Hispanic and Non Hispanic), if I understood correctly.

Reply:
Oh, I thought the way you were suggesting was that Hispanic/Non-hispanic would be part of the "race" column? I may have forgotten the details here...

Reply:
Renamed to "race"